9801-logger/9801-logger.jl

```diff
@@ -169,149 +196,170 @@ try
     println("  Run Time     => $RUN_TIME seconds $is_runtime_default")
     println("  Volt Min     => $VOLT_MIN volts $is_volt_min_default")
     println("  Volt Max     => $VOLT_MAX volts $is_volt_max_default")
-    if (!NO_CSV)
+    if (!NO_CSV && !AUTOMATED)
       println("  Output File  => $csv_name $is_name_default")
     end
 
-    global spin_lock
-    iter_count   = 0
-    recordings   = []
-    timestamps   = []
-    times        = []
-    volts        = []
-    volts_bounds = []
-    freqs        = []
-    reals        = []
-    apps         = []
-    pfacs        = []
-    currs        = []
-    apks         = []
+    
+    test_count   = 1
+    while (test_count <= TEST_RUNS)
```

Comment:
There is no need to have `test_count` and `test_runs` variables. just use a for loop on `tests`